### PR TITLE
Server initial setup

### PR DESCRIPTION
**Pull request by pranavacharya**
1. Initialized Express 
2. Added .gitignore file
3. Added Es-lint (Airbnb style guide )
4. Added nodemon dev dependency and dev script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-  "description": "Rekrus recruitment planner",
-  "main": "index.js",
+  "version": "0.0.0",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node ./bin/www",
+    "dev": "nodemon ./bin/www"
   },
-  "author": "Rekrus",
```
Comment (rohit-ambre) [inline package.json:9]: add author and contributors

File: app.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+var createError = require('http-errors');
```
Comment (rohit-ambre) [inline app.js:1]: use `const` in place of `var`

Comment (pranavacharya) [reply app.js]: Replaced var with const

**Comment by rohit-ambre** [comment]
add eslintignore file

**Comment by pranavacharya** [comment]
Added eslint ignore file

**Pull request merged**